core/templates/dev/head/pages/admin-page/admin-page.module.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the admin page.
+ */
+
+import { NgModule } from '@angular/core';
+
+import { ActivitiesTabModule } from
+  'pages/admin-page/activities-tab/activities-tab.module.ts';
+import { ConfigTabModule } from
+  'pages/admin-page/config-tab/config-tab.module.ts';
+import { AdminJobsTabModule } from
+  'pages/admin-page/jobs-tab/jobs-tab.module.ts';
+import { MiscTabModule } from
+  'pages/admin-page/misc-tab/misc-tab.module.ts';
+import { NavbarModule } from
+  'pages/admin-page/navbar/admin-navbar.module.ts';
+import { RolesTabModule } from 'pages/admin-page/roles-tab/roles-tab.module.ts';
+
+@NgModule({
+  imports: [
+    ActivitiesTabModule,
+    ConfigTabModule,
+    AdminJobsTabModule,
+    MiscTabModule,
+    NavbarModule,
+    RolesTabModule
+  ]
+})
+export class AdminPageModule {}
+
+require('pages/admin-page/activities-tab/activities-tab.module.ts');
+require('pages/admin-page/config-tab/config-tab.module.ts');
+require('pages/admin-page/jobs-tab/jobs-tab.module.ts');
+require('pages/admin-page/misc-tab/misc-tab.module.ts');
+require('pages/admin-page/navbar/admin-navbar.module.ts');
+require('pages/admin-page/roles-tab/roles-tab.module.ts');
+
+angular.module('adminPageModule', [
+  'activitiesTabModule',
+  'configTabModule',
+  'jobsTabModule',
```

Comment:
adminJobsTabModule. Try to match names with imports, here and elsewhere.

Try to also keep the order the same, unless there's a special reason not to do so. E.g. here your rolesTabModule and miscTabModule are switched. And adminNavbarModule is named differently from NavbarModule.

Please check the remaining files too.

Reply:
I had these names because it says [here](https://angular.io/guide/styleguide#angular-ngmodule-names) that the name of the module be after the feature or the folder it resides in. Now, the navbar is something whose folder name is common in many pages, but the others not. And hence this naming. The general pattern followed was to name the module after the folder and if it is a common name then prefix the page name.
Does that sound good?

Reply:
I don't think I mind what you name it, as long as we have a consistent convention. But I think the names should be consistent between the two lists. Is there something in the upgrade guide that suggests that that should not be the case?

Reply:
Oh, my bad I have fixed that. 
The imports are actually alphabetical. So their order and the order in the list may differ. Although the two lists must be the same.
Thanks for pointing that out!